api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -251,6 +279,29 @@ public void alertUserInitialCreditsExpiring(DbUser user) throws MessagingExcepti
         htmlMessage);
   }
 
+  @Override
+  public void alertUserInitialCreditsExpired(DbUser user) throws MessagingException {
+    if (!checkEnabledInitialCreditsExpirationFlag()) {
+      // This is a safety check to avoid sending expired emails when the feature flag is off.
+      log.info("Initial credits expiration feature flag is disabled. Not sending expired email.");
+      throw new BadRequestException("Initial credits expiration feature flag is disabled.");
```

Comment:
Not sure if we want to throw an exception here. If a user's credits are expired and the flag isn't on, I'd think we'd just not send an email rather than fail here